operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ServerPod.java

```diff
@@ -213,6 +213,17 @@
       + "See `kubectl explain pods.spec.containers.volumeMounts`.")
   private final List<V1VolumeMount> volumeMounts = new ArrayList<>();
 
+  /**
+   * The common mount.
+   *
+   */
+  @Description("Use a common mount to automatically include directory content from additional images. "
+          + "This is a useful alternative for including Model in Image model files, or other types of files, in a pod "
+          + "without requiring modifications to the pod's base image 'domain.spec.image'. "
+          + "This feature internally uses a Kubernetes emptyDir volume and Kubernetes init containers to share "
+          + "the files from the additional images with the pod.")
+  private CommonMount commonMount;
```

Comment:
> Overall LGTM, can you verify upgrade won't cause any rolling?

I ran the upgrade tests from 3.1.4 to the latest branch (including my changes) and it didn't cause a rolling restart. I believe Russ has added unit tests to catch the unexpected rolling issue and I believe we should be good here.

Reply:
Regarding the validation error "when a commonMount is defined but no containers are defined within it", I have added validation checks and made the change to not set the common mount path env var. After this check, it won't fail with the surprising error since env var will not be set when no containers are defined. Thanks.

Reply:
Can you point me to the code where you added validation checks? I somehow missed it.

Reply:
The check is in 3 different files (1) JobStepContext.java, (2) JobHelper.java and (3) PodStepContext.java. I added below check to see if commonMount.getContainers is not null using `Optional.ofNullable(cm.getContainers()).ifPresent()` -

```
  protected V1PodSpec addCommonMountInitContainerAndVolume(V1PodSpec podSpec, CommonMount cm) {
    Optional.ofNullable(cm.getContainers()).ifPresent(cl -> addInitContainerAndVolume(podSpec, cm, cl));
    return podSpec;
  }			
```
```
   Optional.ofNullable(cm.getContainers()).ifPresent(containerList ->
            addEnvVar(vars, CommomMountEnvVars.COMMON_MOUNT_PATH, cm.getMountPath()));					
```

Reply:
Sorry - there was a bit of terminology confusion. 

I generally assume a 'validation check' in the operator context means that the operator will generate an error message that's reflected in the status. But no error message is generated here, which might be fine. 

It looks like the operator now silently ignores the case where a commonMount is configured with no containers? 

I'm OK with this - it's better than generating error messages that don't point out the underlying problem  - but wonder if it'd be even better to generate a validation error (plus document that at least one container must be specified). This is similar to the way we error out when 'model' is specified but its required 'model.runtimeEncryptionSecret' setting is missing.

Reply:
ok, I have added the logic to generate a validation error message when no containers are defined. Thanks.